From ff44f9a2c7bd1a38a7edbff70f8401b179e77d71 Mon Sep 17 00:00:00 2001
From: ritsuka <ritsuka@b64f7644-9d1e-0410-96f1-a4d463321fa5>
Date: Fri, 30 Jan 2015 09:21:47 +0000
Subject: [PATCH 01/16] Add a default font directory for fontconfig to use when
 there isn't a configuration file available. Move the font cache to the user
 directory.

git-svn-id: svn://svn.handbrake.fr/HandBrake/trunk@6828 b64f7644-9d1e-0410-96f1-a4d463321fa5
(cherry picked from commit c50d645b3f9049fc99e5c5504813c8ee4396e8ee)
---
 contrib/fontconfig/module.defs | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/contrib/fontconfig/module.defs b/contrib/fontconfig/module.defs
index b527731..0bf9421 100644
--- a/contrib/fontconfig/module.defs
+++ b/contrib/fontconfig/module.defs
@@ -86,7 +86,9 @@ FONTCONFIG.CONFIGURE.extra += \
     LIBXML2_LIBS="-L$(call fn.ABSOLUTE,$(CONTRIB.build/))lib -lxml2" \
     LIBXML2_CFLAGS="-I$(call fn.ABSOLUTE,$(CONTRIB.build/))include/libxml2" \
     --with-fcpath=/usr/X11/lib/X11/fontconfig \
-    --with-cache-dir=/usr/X11/var/cache/fontconfig
+    --with-cache-dir=~/Library/Caches/fontconfig \
+    --with-default-fonts=/Library/Fonts \
+    --with-add-fonts=/System/Library/Fonts,~/Library/Fonts
 else ifeq (linux,$(BUILD.system))
 FONTCONFIG.CONFIGURE.extra += \
     --with-fcpath=/etc/fonts \
-- 
2.3.0

